app/app.js

```diff
@@ -4,6 +4,7 @@
 /* eslint-disable global-require*/
 import {AsyncStorage, Linking, NativeModules} from 'react-native';
 import {setGenericPassword, getGenericPassword, resetGenericPassword} from 'react-native-keychain';
+import 'babel-polyfill';
```

Comment:
can you make it so this is loaded only for iOS and optionally when iOS is below version 10?

Also add a comment so people know why it is still needed